src/modules/core/components/MembersList/Actions/MemberActions.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { defineMessages } from 'react-intl';
+import classnames from 'classnames';
+
+import Icon from '~core/Icon';
+import Popover from '~core/Popover';
+import { Colony } from '~data/index';
+
+import MemberActionsPopover from './MemberActionsPopover';
+
+import styles from './MemberActions.css';
+
+const MSG = defineMessages({
+  memberActionsTitle: {
+    id: 'core.MemberList.MemberActions.MemberActionTitle',
+    defaultMessage: 'Member Actions',
+  },
+});
+
+interface Props {
+  userAddress: string;
+  colony: Colony;
+  isWhitelisted: boolean;
+  isBanned: boolean;
+  canAdministerComments?: boolean;
+}
+
+const displayName = 'core.MemberList.MemberActions';
+
+const MemberActions = ({
+  canAdministerComments,
+  colony,
+  userAddress,
+  isWhitelisted,
+  isBanned,
+}: Props) => {
+  const [isOpen, setOpen] = useState(false);
+  return (
+    <Popover
+      content={({ close }) => (
+        <MemberActionsPopover
+          closePopover={close}
+          canAdministerComments={canAdministerComments}
+          colony={colony}
+          isWhitelisted={isWhitelisted}
+          isBanned={isBanned}
+          userAddress={userAddress}
+        />
+      )}
+      trigger="click"
+      showArrow={false}
+      placement="right"
+      isOpen={isOpen}
+      onClose={() => setOpen(false)}
+      popperOptions={{
+        modifiers: [
+          {
+            name: 'offset',
+            options: {
+              offset: [40, 15],
+            },
+          },
+        ],
+      }}
+    >
+      {({ ref, id }) => (
+        <button
+          id={id}
+          ref={ref}
+          className={classnames(styles.actionsButton, {
+            [styles.activeDropdown]: isOpen,
+          })}
+          onClick={() => setOpen(true)}
+          type="button"
+        >
+          <Icon
+            className={styles.actionsIcon}
+            name="three-dots-row"
+            title={MSG.memberActionsTitle}
+          />
+        </button>
```

Comment:
I think the 3 dots button should be a core component.

Please open a issue that will extract this, especially since we are using it increasingly more places.

Reply:
Issue raised